_rules/video-alternative-for-visual-c5a4ea.md

```diff
@@ -155,7 +155,7 @@ A video element that describes some of the text on the same page. The text on th
 
 #### Failed Example 1
 
-A video element with an incorrect audio description.
+This 'video' element has an incorrect audio description.
```

Comment:
```suggestion
This `video` element has an incorrect audio description.
```
Text enclosed in backticks is rendered as code, which is what we want here as we refer to the name of the HTML element.
('video' vs `video`)